src/resources/ga4gh/variants.proto

```diff
@@ -0,0 +1,241 @@
+/**
+ * This file has been copied from the schema repository of the GA4GH Data Working
+ * Group (https://github.com/ga4gh/ga4gh-schemas). The original file URL is:
+ * https://github.com/ga4gh/ga4gh-schemas/blob/master/src/main/proto/ga4gh/variants.proto
+ * The GA4GH schemas are covered by the Apache license v2.0.
+ */
+
+// This file defines the objects used to represent variant calls, most importantly
+// VariantSet, Variant, and Call.
+// See {TODO: LINK TO VARIANTS OVERVIEW} for more information.
+
+syntax = "proto3";
+
+option java_package = "org.genomicsdb.ga4gh";
+
+package ga4gh;
+
+import "common.proto";
+import "google/protobuf/struct.proto";
+
+// This metadata represents VCF header information.
+message VariantSetMetadata {
+  // The top-level key.
+  string key = 1;
+
+  // The value field for simple metadata.
+  string value = 2;
+
+  // User-provided ID field, not enforced by this API.
+  // Two or more pieces of structured metadata with identical
+  // id and key fields are considered equivalent.
+  // FIXME: If it's not enforced, then why can't it be null?
+  string id = 3;
+
+  // The type of data.
+  string type = 4;
+
+  // The number of values that can be included in a field described by this
+  // metadata.
+  string number = 5;
```

Comment:
should this be int?

Reply:
I just copied this from the GA4GH repo. The metadata is not something I'm looking at right now. Only the Variant and Call objects.